app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,12 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title><%= Rails.application.class.module_parent_name %></title>
+    <title>
+      <%- if content_for(:title) %>
+        <%= content_for(:title) %> |
+      <%- end %>
+      <%= Setting.html_title_base %>
+    </title>
```

Comment:
is there a reason these two title conditions are expressed with two different code styles?